editor/components/rich-text/tinymce.js

```diff
@@ -116,13 +116,6 @@ export default class TinyMCE extends Component {
 		return false;
 	}
 
-	configureIsPlaceholderVisible( isPlaceholderVisible ) {
-		const isPlaceholderVisibleString = String( !! isPlaceholderVisible );
-		if ( this.editorNode.getAttribute( IS_PLACEHOLDER_VISIBLE_ATTR_NAME ) !== isPlaceholderVisibleString ) {
-			this.editorNode.setAttribute( IS_PLACEHOLDER_VISIBLE_ATTR_NAME, isPlaceholderVisibleString );
-		}
-	}
-
 	componentWillReceiveProps( nextProps ) {
```

Comment:
`componentWillReceiveProps` is still here, but I see `configureIsPlaceholderVisible` moved. Is it intentional? Should we use `componentDidUpdate` in here?

Reply:
Commented about this one on the PR description. No clear alternative here (the Fill one as well).